ods_tools/oed/exposure.py

```diff
@@ -229,3 +244,96 @@ def check(self, validation_config=None):
             validation_config = self.validation_config
         validator = Validator(self)
         return validator(validation_config)
+
+    def to_version(self, version):
+        """
+        goes through location to convert columns to specific version.
+        Right now it works for OccupancyCode and ConstructionCode.
+        (please note that it only supports minor version changes in "major.minor" format, e.g. 3.2, 7.4, etc.)
+
+        Args:
+            version (str): specific version to roll to
+
+        Returns:
+            itself (OedExposure): updated object
+        """
+
+        def version_to_tuple(version_str):
+            """Converts the version in string format to a tuple of integers"""
+            return tuple(map(int, version_str.split('.')))
+
+        def strip_version(version_str):
+            """
+            Strips a version string to the format 'major.minor'.
+
+            Args:
+                version_str (str): The version string.
+
+            Returns:
+                str: The stripped version string.
+            """
+            # Use regex to find 'major.minor' format
+            match = re.match(r"(\d+\.\d+)", version_str)
+            if match:
+                return match.group(1)
+            else:
+                raise ValueError(f"Invalid version format: {version_str}")
+
+        # Strip the version string down to 'major.minor' format
+        stripped_version = strip_version(version)
+
+        # Input checks - format, version, and convert to tuple
+        if not re.match(r"^\d+\.\d+$", stripped_version):
+            raise ValueError(
+                "Version should be provided in 'major.minor' format, e.g. 3.2, 7.4, etc."
+            )
+
+        if stripped_version not in self.oed_schema.schema["versioning"]:
+            raise ValueError(
+                f"Version {version} is not a known version present in the OED schema."
+            )
+        try:
+            version_tuple = version_to_tuple(stripped_version)
+        except ValueError:
+            raise ValueError(f"Version {stripped_version} is not a valid number.")
+
+        # TODO: Determine the current version. If the current version is the same as the target version, return the current object.
```

Comment:
The rule we try to follow is no TODO in the code when released, if something is left to be done, we keep track of it via git issue

Reply:
Removed! Probably it is not TODO anymore, given the fact that, if version passed > all versions in the list, nothing will happen.